### DISCUSSION TITLE
create custom puzzles

### DISCUSSION THREAD

**Discussion by meinradr** [michel-kraemer/sudocle#247] [Q&A] [answered]
How can I set my own puzzles?

**Comment by michel-kraemer** [comment, marked as answer]
Sudocle does not provide the possibility to set your own puzzles (yet) but you can do the following:
1) Create your puzzle in f-puzzles: https://f-puzzles.com/
2) Press the "Export" button and select "Open With Link"
3) A new tab will open in your browser. The URL in your browser's address bar will look something like this:
    https://f-puzzles.com/?load=N4IgzglgXgpiBcBOANCA5gJwgEwQbT2AF9ljSSzKLryBdZQmq8l54+x0kANwEMAbAK5x4ARlRoI3GADsEAFwzCmK1tQ5tVmug216VGrRR4DhCAEwSpshUphGdnfWrWGXzqm49MTQkQGYraTl4RWU6WiIgA==
4) Copy the ID (basically everything after `load=`), prepend the string `fpuzzles` to it, and then append to the URL of your Sudocle instance. For example, if you're using Sudocle on my domain, the URL should look as follows:
    https://michelkraemer.com/sudocle/fpuzzlesN4IgzglgXgpiBcBOANCA5gJwgEwQbT2AF9ljSSzKLryBdZQmq8l54+x0kANwEMAbAK5x4ARlRoI3GADsEAFwzCmK1tQ5tVmug216VGrRR4DhCAEwSpshUphGdnfWrWGXzqm49MTQkQGYraTl4RWU6WiIgA==
